lib/msal-browser/docs/logout.md

```diff
@@ -59,21 +59,6 @@ await msalInstance.logoutRedirect({
 });
 ```
 
-### Skipping the server sign-out
```

Comment:
We don't need to remove this section, just update it to use the PCA config option instead